devops/windows/base-docker-wsl/helper/log/logs-jempi-input-disi.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+set -u
+
+source ../../0-conf.env
+docker service logs --raw ${STACK_NAME}_input-disi
```

Comment:
What is the `input-disi` service?

Reply:
this script can be deleted --- also in for the docker environment